helpers/mathjax.js

```diff
@@ -83,9 +59,10 @@ export function createHtmlBlockRenderer() {
 
 export function loadMathJax(mathJaxConfig) {
 
-	if (mathJaxLoaded) return mathJaxLoaded;
+	const win = (mathJaxConfig && mathJaxConfig.window) || window;
+	if (win.mathJaxLoaded) return win.mathJaxLoaded;
```

Comment:
Nit: usually we try to avoid putting something non-D2L specific in the global namespace. FWIW, [we do this for MathJax in BSI](https://github.com/Brightspace/brightspace-integration/blob/main/web-components/mathjax.js#L4). I would at least prefix the variable with d2l to avoid chances of a collision with MathJax itself.

Reply:
Yeah, I can prefix this. Or I can create `win.D2L` and use that structure. I'm fine either way.

Reply:
I opted for the latter for consistency.